src/config/mod.rs

```diff
@@ -135,7 +135,7 @@ create_config! {
         "Report all, none or unnumbered occurrences of TODO in source file comments";
     report_fixme: ReportTactic, ReportTactic::Never, false,
         "Report all, none or unnumbered occurrences of FIXME in source file comments";
-    ignore: IgnoreList, IgnoreList::default(), false,
+    ignore: IgnoreList, IgnoreList::default(), true,
```

Comment:
Would you please not make `ignore` stable in this PR? I would prefer not to mark it stable until it gets tested a bit more.